ko_KR/readme.md

```diff
@@ -0,0 +1,971 @@
+
+```js
+test.before.skip(...);
+test.skip.after(...);
+test.serial.only(...);
+test.only.serial(...);
+```
+
+이 말은 임시적으로 `.skip` 또는 `.only`를 테스트의 끝에 추가하거나 훅 선언에서 별 다른 수정 없이 사용할 수 있다는 말입니다.
+
+### 커스텀 가정
+
+내장에서 제공하는 가정 라이브러리 대신, 또는 추가적으로 가정 실패시 예외를 던지는것을 지원한다면, 어느 가정 라이브러리라도 사용할 수 있습니다.
+
+이는 [빌트-인 가정](#가정)을 사용하는 것과는 달리 좋은 경험을 제공하지 못하며 [가정 계획](#가정-계획하기)을 사용할 수 없습니다. ([이슈 #25](https://github.com/sindresorhus/ava/issues/25)를 참고하세요)
+
+```js
+import assert from 'assert';
+
+test(t => {
+	assert(true);
+});
+```
+
+### ES2015 지원
+
+AVA는 [Babel 6](https://babeljs.io)를 통한 ES2015를 내부적으로 기본 지원합니다. 그저 테스트 코드를 ES2015로 작성하면 됩니다. 다른 추가 작업은 필요하지 않습니다. 또한 어떠한 Babel 버전도 프로젝트 내에서 사용할 수 있습니다. 내부에서 자체적으로 [`es2015`](https://babeljs.io/docs/plugins/preset-es2015/)와 [`stage-2`](https://babeljs.io/docs/plugins/preset-stage-2/) 프리셋을 사용함과 동시에 [`espower`](https://github.com/power-assert-js/babel-plugin-espower)와 [`transform-runtime`](https://babeljs.io/docs/plugins/transform-runtime/) 플러그인을 사용합니다.
+
+AVA에서 사용하는 Babel 구성은 다음과 같습니다:
+
+```json
+{
+  "presets": [
+    "es2015",
+    "stage-2"
+  ],
+  "plugins": [
+    "espower",
+    "transform-runtime"
+  ]
+}
+```
+
+AVA의 [`package.json` 구성](#구성)에서 `babel` 옵션을 통해 AVA 테스트 파일의 트랜스파일 동작을 커스터마이즈 할 수 있습니다. 예를 들어 프리셋을 덮어쓰려면 다음과 같이 설정할 수 있습니다:
+
+```json
+{
+	"ava": {
+		 "babel": {
+			 "presets": [
+					"es2015",
+					"stage-0",
+					"react"
+			 ]
+		 }
+	},
+}
+```
+
+또한 특별한 `"inherit"` 키워드를 사용할 수 있습니다. 이 키워드는 AVA가 프로젝트 내의 [`.babelrc` 또는 `package.json` 파일](https://babeljs.io/docs/usage/babelrc/)을 사용하도록 구성을 연기합니다. 이 방법을 사용하면 AVA를 위해 추가적인 설정을 할 필요 없이 소스 파일에 대한 구성을 그대로 사용하여 테스트 파일을 트랜스파일 할 수 있습니다:
+
+ ```json
+{
+	"babel": {
+		"presets": [
+			"es2015",
+			"stage-0",
+			"react"
+		]
+	},
+	"ava": {
+		"babel": "inherit"
+	},
+}
+```
+더 많은 예시와 자세한 구성 옵션에 대한 설명은 AVA의 [`.babelrc` 레시피](docs/recipes/babelrc.md)를 참고하세요.
+
+참고로 AVA는 구성에 상관없이 *언제나* [내부적으로 몇 가지 플러그인](docs/recipes/babelrc.md#notes)을 적용합니다. 하지만 실제 코드에 영향을 줄 가능성은 적습니다.
+
+### TypeScript 지원
+
+AVA는 TypeScript의 타이핑을 포함하고 있습니다. 하지만 직접적인 설정이 필요합니다. `tsconfig.json` 파일에서 `module`을 `commonjs`로 설정하면, TypeScript는 자동으로 AVA에 대한 타입 정의를 찾을 것입니다. Promise와 비동기 함수를 사용하려면 `target`을 `es2015`로 설정해야 합니다.
+
+### 들여온 모듈 트랜스파일하기
+
+현재로써 AVA는 지정한 테스트 파일만 트랜스파일합니다. *따라서 테스트 밖에서 `import`로 들여오는 모듈은 트랜스파일하지 않습니다.* 이러한 상황은 예상치 못한 상황이겠지만, 해결 방법이 있습니다.
+
+만약 Babel을 사용한다면 [require 훅](https://babeljs.io/docs/usage/require/)을 사용하여 들여오는 모듈을 즉석에서 트랜스파일 할 수 있습니다. AVA를 `--require babel-register` 옵션 ([CLI 사용법](#cli-사용법) 참고)과 함께 실행하거나 [`package.json`에서 설정](#구성)할 수 있습니다.
+
+또한 별도의 프로세스에서 모듈을 트랜스파일할 수도 있으며 테스트에서 소스를 가져오는 대신 트랜스파일된 파일을 참조할 수도 있습니다.
+
+### Promise 지원
+
+테스트에서 promise를 반환하는 경우 promise가 resolve될 때 자동으로 테스트가 끝나므로 따로 명시적으로 끝낼 필요가 없습니다.
+
+```js
+test(t => {
+	return somePromise().then(result => {
+		t.is(result, 'unicorn');
+	});
+});
+```
+
+### 생성기 함수 지원
+
+AVA는 내부적으로 [생성기 함수](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*)를 지원합니다.
```

Comment:
korean link is better, replace:
`(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*)`
by:
`(https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Statements/function*)`